assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,33 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => param;
```

Comment:
The parens are optional if you only have one parameter in the function signature. However some developers still add it to distinguish clearly between the parameters and the return values (sometimes its easy to get confused between `=>` and `=` - it has happened to me) and also it makes it convenient to add more parameters later.